upgrade/helpers/tasks.py

```diff
@@ -799,3 +799,33 @@ def pre_upgrade_system_checks(capsules):
     if capsules:
         for capsule in capsules:
             wait_untill_capsule_sync(capsule)
+
+
+def generate_custom_certs():
+    """ Task to generate custom certs for satellite
+    :param: SERVER_HOSTNAME
+    :return: custom certs
+    """
+    certs_script = StringIO()
+    certs_script.write(
+        u"#! /bin/bash\n"
+        u"name={hostname}\n"
+        u"mkdir ownca\n"
+        u"pushd ownca\n"
+        u"wget https://raw.githubusercontent.com/ntkathole"
```

Comment:
See if you can make it in 99 words as now we have changed that setting in .flake8

Applicable for all other lines.